magnet_derive/src/codegen_field.rs

```diff
@@ -51,10 +51,17 @@ fn impl_bson_schema_named_fields(
 ) -> Result<TokenStream> {
     let properties = &field_names(attrs, &fields)?;
     let defs: Vec<_> = fields.iter().map(field_def).collect::<Result<_>>()?;
+    let doc = doc_meta(&attrs).and_then(|doc| meta_value_as_str(&doc).ok());
+    let doc = if doc.is_some() {
+        quote! { "description": #doc.trim_left(), }
+    } else {
+        quote! {}
```

Comment:
I think this would be better served by

    doc.map(|doc| quote! { "description": #doc.trim_left(), }).unwrap_or_default()

Also, the trailing comma should be at the use site for better readability.

Reply:
hm, but if #doc is empty it would error out for extra comma

Reply:
oh, that is quite right. I didn't notice that. Nevermind, then. :)

Reply:
However, that should be true for the other similar places, then. For example, when `doc == None`, then this would be also be a syntax error: [codegen_enum.rs:19-22](https://github.com/H2CO3/magnet/pull/2/files#diff-41e1b1a45d20820cbead3c7ff8d87ed3R19)